<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>787</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Rubik</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	background-color: rgb(234, 234, 234);
	border:3px solid rgb(167, 167, 167);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_name">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>591</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rubik</family>
      <pointsize>47</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Счётчик Файлов</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_fayl">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>240</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ваши файлы : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_fa">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>160</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Выберите Файлы</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>300</y>
      <width>441</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="isWrapping" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pre">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>660</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Преобразовать файлы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_och">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>660</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Очистить список</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_kat">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Выберите Категорию</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_kat">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Категория : </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>600</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 14px</string>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_del">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>600</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Удалить выбранные</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_otschet">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>600</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Начать отсчёт с :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_down">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Перенести на 1 вниз</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_up">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Перенести на 1 вверх</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
